packages/starknet-snap/src/rpcs/execute-txn.ts

```diff
@@ -146,8 +146,10 @@ export class ExecuteTxnRpc extends AccountRpcController<
       interfaceId: '',
       type: TransactionType.INVOKE,
       signer: address,
+      addressIndex: this.account.addressIndex,
       maxFee: suggestedMaxFee,
       calls: formattedCalls,
+      resourceBounds: estimateResults.map((result) => result.resourceBounds),
```

Comment:
im not sure if i getting wrong or not, it seem missing to pull back the est result to below for execute txn